### PR TITLE
Clarify what needs to be done to install this

### LINKED ISSUE #3: Incorrect / Invalid Brew Installation Instructions
After attempting to follow your installation guide, this isn't working for me:

``` shell
~ ❯❯❯ boot2docker up                                                          ⏎
Waiting for VM and Docker daemon to start...
.o
Started.
Writing /Users/kkirsche/.boot2docker/certs/boot2docker-vm/ca.pem
Writing /Users/kkirsche/.boot2docker/certs/boot2docker-vm/cert.pem
Writing /Users/kkirsche/.boot2docker/certs/boot2docker-vm/key.pem

To connect the Docker client to the Docker daemon, please set:
    export DOCKER_HOST=tcp://192.168.59.103:2376
    export DOCKER_CERT_PATH=/Users/kkirsche/.boot2docker/certs/boot2docker-vm
    export DOCKER_TLS_VERIFY=1

~ ❯❯❯ boot2docker shellinit
Writing /Users/kkirsche/.boot2docker/certs/boot2docker-vm/ca.pem
Writing /Users/kkirsche/.boot2docker/certs/boot2docker-vm/cert.pem
Writing /Users/kkirsche/.boot2docker/certs/boot2docker-vm/key.pem
    export DOCKER_HOST=tcp://192.168.59.103:2376
    export DOCKER_CERT_PATH=/Users/kkirsche/.boot2docker/certs/boot2docker-vm
    export DOCKER_TLS_VERIFY=1
~ ❯❯❯ brew install codeclimate
==> Installing codeclimate from codeclimate/homebrew-formulae
==> Downloading https://github.com/codeclimate/codeclimate/archive/v0.0.6.tar.gz
==> Downloading from: https://codeload.github.com/codeclimate/codeclimate/tar.gz
Already downloaded: /Library/Caches/Homebrew/codeclimate-0.0.6.tar.gz
==> env PATH=/usr/local/bin:/usr/local/Library/ENV/4.3:/usr/bin:/bin:/usr/sbin:/
bin/check
Get http:///var/run/docker.sock/v1.19/version: dial unix /var/run/docker.sock: no such file or directory. Are you trying to connect to a TLS-enabled daemon without TLS?
Unable to run `docker version', the docker daemon may not be running
Please ensure `boot2docker up' succeeds and you've run `boot2docker shellinit' in this shell
make: *** [install] Error 1

READ THIS: https://git.io/brew-troubleshooting
If reporting this issue please do so at (not Homebrew/homebrew):
  https://github.com/codeclimate/homebrew-formulae/issues
```

### PR DESCRIPTION
**Pull request by kkirsche**
Fix #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
 
 ## Usage
 
+Ensure docker is running or if you are using Boot2Docker, run:
+
+```console
+boot2docker up
+boot2docker shellinit
```
Comment (brynary) [inline README.md]: I think this line needs to encapsulated in backticks

Comment (kkirsche) [inline README.md]: @brynary — Why would clarifying how it is run be in back ticks? There is no code in that line which is being executed by the user.

Comment (pbrisbin) [inline README.md]: <del>My understanding about `shellinit` was that it outputs clear messaging about environment variables you should set in your shell profile.</del>


_EDIT_: what I was told about the `shellinit` command doesn't seem to be accurate. I agree that we want to use the `eval` form that actually sets the variables for you. Backticks are an old deprecated construct though, what you want is this:

```
eval $(boot2docker shellinit)
```

Comment (pbrisbin) [inline README.md]: The boot2docker README does use the `eval` line without further explanation, so we should probably follow that.

Comment (kkirsche) [inline README.md]: The eval version while setting it for the current shell session doesn't solve the problem though of it not being in the profile files and loading automatically in the future

**Comment by pbrisbin** [comment]
Awesome, thanks! I'm going to go ahead and merge but then re-organize the README a little to make this convey that it's specific to the codeclimate formula present (it's just coincidence that that's the _only_ formula currently here), not all of them may require docker and this setup.

**Pull request merged**